tiatoolbox/models/dataset/dataset_abc.py

```diff
@@ -3,17 +3,29 @@
 
 from abc import ABC, abstractmethod
 from pathlib import Path
-from typing import Callable
+from typing import TYPE_CHECKING, Callable, Iterable, List, Union
+
+if TYPE_CHECKING:
```

Comment:
Please use #pragma no cover on this line as this does not need to be tested for coverage.